app/controllers/todo_list_controller.rb

```diff
@@ -38,7 +39,7 @@ def create
     end
 
     def update        
-        if TodoList.find(params[:id]).update(post_params.merge(:id_usuario => current_user.id, :done_homework => 0))
+        if TodoList.find(params[:id]).update(post_params.merge(:id_usuario => current_user.id))
```

Comment:
you're using the old ruby syntax for hashes by using the `=>`, can you update it, please